dist/s_string

```diff
@@ -45,9 +45,12 @@ check() {
 }
 
 # List of files to spellchk.
+# FIXME-WT-9433 Skip files in the test/cppsuite folder.
 l=`(cd .. &&
-    find bench examples ext src test -name '*.[chsy]' &&
-    find src -name '*.in' && find test -name '*.cxx' && find test -name '*.cpp')`
+    find bench examples ext src test -not -path "test/cppsuite/*" -name '*.[chsy]' &&
```

Comment:
Can you also add `-not -path "ext/storage_sources/s3_store/*"`? 
That way it’s explicit we also need to check the s3 code when fixing WT-9433